{"name": "test_wrong_password", "status": "passed", "description": "\"test_wrong_password\" is the test that try auth with wrong password", "steps": [{"name": "authorization wrong password", "status": "passed", "start": 1694858084778, "stop": 1694858091099}], "attachments": [{"name": "stdout", "source": "65ec2f11-8996-4bff-940c-30835b88a569-attachment.txt", "type": "text/plain"}], "start": 1694858080587, "stop": 1694858091233, "uuid": "c5d4c74e-ffa0-4bc7-9241-608e94a4d468", "historyId": "b0cebe0be7821dec708730b8b3e41e3e", "testCaseId": "b0cebe0be7821dec708730b8b3e41e3e", "fullName": "tests.test_authorizations#test_wrong_password", "labels": [{"name": "tag", "value": "order(2)"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_authorizations"}, {"name": "host", "value": "MacBook-Air-Aleksej.local"}, {"name": "thread", "value": "18514-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_authorizations"}]}